test/controllers/EditSetControllerSpec.scala

```diff
@@ -1701,6 +1702,47 @@ class EditSetControllerSpec extends BaseSpec {
 
           }
         }
+        "background" when {
+          "is too long" in {
+
+            val excessivelyLongBackground = "Something about one of the people." * 400
+            val values = validValuesForSaving ++ Map("background" -> excessivelyLongBackground)
+
+            val result = submitWhileLoggedIn(1, "COAL.2022.V5RJW.P", values)
```

Comment:
@tna-erasmos for consistency this record id needs to match with the expected record id  `COAL.2022.V1RJW.P`

Reply:
Changed.

Reply:
LGTM